### ISSUE TITLE
Design v1 Status Tracking Experience

### ISSUE THREAD

**Issue by quarterback** [18F/cms-hitech-apd#400]
When APDs are submitted, what does an analyst see in the tracking progress? A lightweight tracking screen might be helpful for checking status after submission.

### This task is done when...
 - [x] A prototype of the design experience is designed

**Comment by quarterback** [comment]
<img width="1244" alt="screenshot 2018-05-25 11 50 49" src="https://user-images.githubusercontent.com/4461062/40553914-243b2cc2-6012-11e8-8701-3323380e764d.png">

Here's a first pass at a status tracker with copy. It's deliberately simple and obviously, I'd imagine the styling would mirror the existing application, I just forked something else to sketch this out.

**Comment by NAretakis** [comment]
This look great!
I've included an adjusted list of the current statuses we use in our internal tracking. Maybe adjust the language to use our current list for clarity?

1. **Received** - this means we have received the submission but not cracked it open for review
2. **Under Review** - this means we have/are reviewing the submission
3. **Response sent to state, on hold for revisions** -  (perhaps this = returned?) this means we have comments/questions that the state must address before we can approve.
4. **Review complete** - this means the submission and review by the analyst has been completed, an approval letter is drafted, and it is now going to enter into the secondary and clearance phases.
5. **Secondary Signoff** - this means that a peer is reviewing the approval letter and submission. 
6. **Approval letter in Clearance** - this means that CMS leadership is reviewing the approval letter decision package.

1-6 above is a progression, with the possibility to repeat steps 3 and 2 several times. The items below are possible forks depending on the ultimate result.

7. **Approved** - submission is approved.
8. **Withdrawn** - this means that the submission did not meet CMS' requirements and the state withdrew the APD.
9. **Denied** - extreme rarity, but we should have a status reflecting the ability to outright deny a submission.

**Comment by jeromeleecms** [comment]
Tagging #350 and #383 for reference + previous wireframes.

@quarterback Tying this back to our previous discussions on this, where do you see this fits for the analyst use? In #350, @lauraponce had mocked up the States at a Glance view - is this complementary, in lieu of, or something different? For context, if this is tracking multiple submissions, it might be helpful to see what this would look conceptually with multiple line items. 

Also, it will probably be worth talking about the user interaction to get these statuses updated (which will inform the copy). Are we going to track the status of the submission and the outcome of the submission as two different fields or would it be one? If they are together, then 6 in @NAretakis list would become "Letter in Clearance" and we can use the branching outcome. If they are separate, then 7-8-9 would merge to a single "Complete" status and the outcome would be as Nick described. Does that make sense?

**Comment by jeromeleecms** [comment]
@quarterback Apologies that I missed the mark on your intent here! 

Your question got me thinking on something I've been noodling over for a little bit (#570) and I was thinking this is not so much of a closure experience as opposed to a broader [workflow question](https://drive.google.com/open?id=0B4B0xeCMEaFyVTkwelVFWlBwTWtMYmd0SGREVmV4T0lCclJR). 

This was an early proposed workflow which is probably way too detailed for your question, but when you mentioned lightweight tracking, I was thinking how to distill the flow down into meaningful pieces so the status in the workflow = what would be displayed in the dashboard (and provide actionable/useful details to the analyst). 

The proposed statuses actually give us less information than we track in our internal tracker now, but I def. agree with where you're mind is at on this. Most of the workflow is still relevant, but I'm still not sure how the system pieces will fit into this and then roll up into this high level view. Your question sent me down a bit of a rabbit hole...

If you need me to talk/walk you through anything, let me know.

**Comment by quarterback** [comment]
@jeromeleecms This is good information and exegesis. Let me noodle with this topic again this week as I settle back in, and will explore the tracking piece more clearer after I check the rest of the queue.

**Comment by quarterback** [comment]
Already working on next iterations of this. Integrated into the dashboard prototypes and so forth, so closing.
